debian/debian/control

```diff
@@ -9,6 +9,7 @@ Homepage: http://www.truenas.com
 Package: truenas
 Architecture: all
 Depends: collectd,
+         cpuid,
```

Comment:
For what reason? commit doesnt say

Reply:
It's used in debug, I forgot to add it when we ported ixdiagnose to linux.